### DISCUSSION TITLE
Can't use links to lines in long files on android / web or mobile app

### DISCUSSION THREAD

**Discussion by tyeth** [community/community#75446] [Mobile]
### Select Topic Area

Bug

### Body

Links are stolen by github mobile, which fail to render the line linked:
https://github.com/espressif/arduino-esp32/blob/master/boards.txt#L10352C51-L10352C51
Disabling link integration for github mobile app stops the first problem, but unfortunately github website on a mobile (chrome or webkit) doesn't support the links either and says the file is too big. It works fine on my laptop in chrome.
On mobile i was able to render the raw file at least.

**Comment by HenSquared** [comment]
Hi @tyeth , sorry this is impacting you! 

As a workaround to the issue on web, you can log in to view the file. Some large files cannot be shown automatically for logged out users. You can also select 'view raw' but that does not show you the selected lines. 

For the native mobile apps, I have opened an issue in the iOS repository. I assume you are on iOS because you mentioned webkit. If you are on Android, let me know and I will duplicate the issue to Android team as well.

**Reply by tyeth** [reply]
Hey @HenSquared thanks for the workarounds, interesting to note the difference for logged in users.
Sorry to say Android user here 😁

**Reply by HenSquared** [reply]
Ah! I guessed wrong. :) 

Thanks for the info :)
